### PR TITLE
Try to fix issue #76

### PR DESCRIPTION
**Pull request by wym6912**
Try to fix issue #76

### CODE REVIEW

File: updatenewcard.py [hunk 1]
```diff
@@ -45,6 +47,7 @@ def namechange(name):
     #cntc = cndbconn.execute('SELECT * FROM unit_m;')
     #cntmp = cntc.fetchone()
     while jptmp:#[0] < 30:
+        print jptmp
```
Comment (wym6912) [inline updatenewcard.py:50]: This line is debug line of printing. Please delete this line.

File: updatenewcard.py [hunk 2]
```diff
@@ -57,59 +60,61 @@ def namechange(name):
...
         if card['support'] == 0 and card['special'] == 0:
-            patterntmp = jpdbconn.execute('SELECT * FROM unit_level_up_pattern_m WHERE unit_level_up_pattern_id = '+str(jptmp[23])+' AND unit_level = '+str(jptmp[18])+';')
+            patterntmp = jpdbconn.execute('SELECT * FROM unit_level_up_pattern_m WHERE unit_level_up_pattern_id = '+str(jptmp[23 + offset])+' AND unit_level = '+str(jptmp[18 + offset])+';')
             tmp = patterntmp.fetchone()
             card['smile'] = card['smile2']-tmp[4]
             card['pure'] = card['pure2']-tmp[5]
             card['cool'] = card['cool2']-tmp[6]
 
         #skill
+        print 'skillid = ' + str(skillid)
```
Comment (wym6912) [inline updatenewcard.py:113]: This line is debug line of printing. Please delete this line.

File: updatenewcard.py [hunk 3]
```diff
@@ -57,59 +60,61 @@ def namechange(name):
...
         if skillid and card['support'] == 0:
             jpskilltmp = jpdbconn.execute('SELECT name,skill_effect_type,trigger_type,unit_skill_level_up_pattern_id FROM unit_skill_m WHERE unit_skill_id = '+str(skillid)+';')
             jpskill = jpskilltmp.fetchone()
+            print 'jpskill = '+ str(jpskill)
```
Comment (wym6912) [inline updatenewcard.py:117]: This line is debug line of printing. Please delete this line.

**Comment by ben1222** [comment]
Fix in #78 , will close this PR

**Pull request closed without merging**